mava/components/updating/parameter_server.py

```diff
@@ -31,6 +31,7 @@
 class ParameterServerConfig:
     non_blocking_sleep_seconds: int = 10
     experiment_path: str = "~/mava/"
+    json_path: Optional[str] = None
```

Comment:
This config param never been used

Reply:
Good catch. It is passed in via the logger config and not a system parameter. I have removed it.